app/helpers/application_helper.rb

```diff
@@ -1,2 +1,17 @@
 module ApplicationHelper
+  def embedded_svg(filename, options = {})
+    assets = Rails.application.assets
+    asset = assets.find_asset(filename)
+
+    if asset
+      file = asset.source.force_encoding("UTF-8")
+      doc = Nokogiri::HTML::DocumentFragment.parse file
+      svg = doc.at_css "svg"
+      svg["class"] = options[:class] if options[:class].present?
+    else
+      doc = "<!-- SVG #{filename} not found -->"
+    end
+
+    raw doc
+  end
```

Comment:
Check out this reference for where this helper method came from: https://blog.cloud66.com/using-svgs-in-a-rails-stack/

This method basically helps us display the SVG files in the views.

Reply:
Niceeee, thank you for posting source!